libraries/lib-mixer/SampleTrackStuff.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "SampleCount.h"
+
+#include <chrono>
+#include <fstream>
+#include <memory>
+#include <vector>
+
+namespace LibSampleTrack
+{
+using BlockPtr = std::shared_ptr<std::vector<float>>;
+using BlockWPtr = std::weak_ptr<std::vector<float>>;
+
+struct MIXER_API ClipBlockSequence
+{
+   // adjacent sample blocks
```

Comment:
Just replacing `//` with `//!` on a brief comment line before a member variable allows
Doxygen to extract the comment.  Do that with each member of this structure and also explain `usefulLength`.